packages/data-lake-api/src/getConnectionConfig.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
+
+import { requireEnv, getEnvVarOrDefault } from '@tupaia/utils';
+
+export const getConnectionConfig = () => ({
+  host: requireEnv('DATA_LAKE_DB_URL'),
+  port: getEnvVarOrDefault('DATA_LAKE_DB_PORT', 5432),
+  user: requireEnv('DATA_LAKE_DB_USER'),
+  password: requireEnv('DATA_LAKE_DB_PASSWORD'),
+  database: requireEnv('DATA_LAKE_DB_NAME'),
+  ssl:
+    process.env.DATA_LAKE_DB_ENABLE_SSL === 'true'
```

Comment:
It seems like we use both DB_ENABLE_SSL and DB_DISABLE_SSL patterns currently... my preference is for the default to be on and you have to pass a flag DISABLE_SSL to turn it off. Not a biggie though.

Reply:
Come to think of it, since nobody will be running the data-lake in local, perhaps we will always be using SSL? I might just scrap this key 👍

Reply:
That works too